GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx

```diff
@@ -133,13 +158,26 @@ const RegistrationForm = () => {
             ReferenceId:pid,
             FamilyMembers: personsList
         }
-        fetch('https://localhost:5001/api/recipient', {
+        setIsLoading(true);
+        await fetch('https://localhost:5001/api/recipient', {
```

Comment:
This should be:  await fetch('api/recipient') instead

Reply:
Yeah that might be a good idea! 👍